### PR TITLE
Fix link to REST API reference docs

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

This fixes https://github.com/ManageIQ/manageiq.org/issues/581

### CODE REVIEW

File: site/docs/api.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ permalink: /docs/api
 title: API Documenation
 ---
 
-This guide provides web services available to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
+On the panel to the left, you will find a detailed overview of REST API, reference material, appendices, and plenty of examples. The guides provide information on how to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
```
Comment (chrisarcand) [inline site/docs/api.md]: "On the panel to the left" doesn't make sense as this text is used in more than one view. For example, the main docs view:



Just say "In this guide you will find a detailed overview of..."

Comment (JPrause) [reply site/docs/api.md]: Cool thanks @chrisarcand will make that change.

File: site/docs/api.md [hunk 2]
```diff
@@ -5,4 +5,4 @@ permalink: /docs/api
 title: API Documenation
 ---
 
-This guide provides web services available to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
+In this guide you will find a detailed overview of REST API, reference material, appendices, and plenty of examples. The guides provide information on how to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
```
Comment (cybette) [inline site/docs/api.md]: Should similar change be made to site/docs/index.md ?

Comment (chrisarcand) [reply site/docs/api.md]: Ah. I thought that the text being the same meant it was sourcing from the same file in my feedback above, which apparently is not the case. @JPrause Looks like you'll indeed want to change the separate docs/index.md too.

Your previous edit is still valid though as when you resize the window down to a mobile view the links still won't be on the left 😄

**Pull request merged**